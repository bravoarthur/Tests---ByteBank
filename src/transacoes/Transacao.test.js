import React from 'react';
import { render } from '@testing-library/react';
import Transacao from './Transacao';



describe('componente de transacao do Extrato', () => {
// O snapShot e como se fosse um print do componente e se por algum motivo algo for motificado nele o test dara erro. Se for mudanca esperada pode apertar 'u' para refazer o snap, caso seja inesperada, corrigir o condigo
    it('O SnapShot do component deve permanecer sempre o mesmo',
     () => {
        const { container } = render(<Transacao
            data='08/09/2020'
            tipo='saque'
            valor='50.00'
        
        />)

        expect(container.firstChild).toMatchSnapshot()


    })


})