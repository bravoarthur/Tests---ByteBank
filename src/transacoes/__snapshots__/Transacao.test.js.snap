// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`componente de transacao do Extrato O SnapShot do component deve permanecer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    08/09/2020
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    50.00
  </p>
</div>
`;
